README.md

```diff
@@ -69,6 +69,12 @@ You can manually add assets in three ways:
 
   This may be the easiest for airdrop claims. Go to the claim site and go through the normal process for claiming. When the site sends the transaction to your wallet for signing - *don't sign* (you probably can't anyway for lack of funds). Instead of signing, copy the raw transaction and paste it in this box. Read this note on [airdrop claims!](#airdrop-claims)
 
+### 4. AbiNinja
+
+  The [AbiNinja](https://abi.ninja) method is great for helping you see what functions are available to call on the contract address you paste in. Paste in an address and then click "Load Contract". Next you should click the hamburger menu to see the read and write functions. Select the write function you want to use and then close the menu. You should see the function with inputs for each parameter required. Once you have entered the parameters you can click "Send" and it should return you to the asset selection view in Hacked Wallet Recovery with your new transaction selected. If you encounter errors while in AbiNinja it may be giving you helpful feedback on a mistake you made when adding a parameter.
```

Comment:
Please feel free to ignore this suggestion, if it adds unnecessary voice because already it feels good but here: 

> Select the write function you want to use and then close the menu.

Just a small nitpick on closing the menu, should we be more verbose like "then close the menu by either clicking outside of sidebar or at very top X icon in sidebar of abi.ninja"

<img width="500" alt="image" src="https://github.com/BuidlGuidl/flashbot-recovery-bundler/assets/80153681/91504aa6-b76e-47fe-9678-0d78211b886d">

saying this because when you scroll down for write function the X icon is hidden a bit but lol again if the closing seems too obvious and already intuitive please feel free to mark it as resolve!